.changeset/whole-toes-add.md

```diff
@@ -0,0 +1,6 @@
+---
+"@workflow/web-shared": patch
+"@workflow/cli": patch
```

Comment:
nit: the cli changeset looks like it's actually just changing alpha to beta in the text

web-shared is the node16 change